pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/participant/Util.java

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.dsm.model.participant;
+
+import org.broadinstitute.dsm.statics.DBConstants;
+
```

Comment:
this code will be cleaner if you make a list of all the possibilities and then use contains for the check.  so
```
public static List<String> DSM_KEY_FIELDS = Arrays.asList(DBConstants.DDP_PARTICIPANT_ALIAS, 
DBConstants.DDP_ONC_HISTORY_DETAIL_ALIAS, DBConstants.DDP_ONC_HISTORY_DETAIL_ALIAS, ...)
```
and then the method can just be
```
public static boolean isUnderDsmKey(String source) {
  return DSM_KEY_FIELDS.contains(source);
}
```